package.json

```
@@ -18,6 +18,12 @@
         "link:monaco-into-playground": "cd web/playground && npm link @abaplint/monaco",
         "unlink-local": "cd web/playground && npm unlink --no-save @abaplint/monaco && npm unlink --no-save @abaplint/core && npm install && cd ../../packages/cli && npm unlink --no-save @abaplint/core && npm install && cd ../monaco && npm unlink --no-save @abaplint/core && npm install"
     },
+    "mocha": {
+        "recursive": true,
+        "reporter": "progress",
+        "spec": "build/test/**/*.js",
```

how does this work? there is no `build` folder in the root. And can it be solved by 

```suggestion
        "spec": "packages/*/build/test/**/*.js",
```

---

Oh, I had an untracked build folder in my workspace from way before and this somehow magically worked off of that. With the right path there seem to be no problems, I will remove the debugging note.